### DISCUSSION TITLE
Calendar: change the photo cover

### DISCUSSION THREAD

**Discussion by sylvaindubus** [photoprism/photoprism#1869] [Q&A] [answered]
Hello!

I wonder how you can change the photo that is displayed as a cover for a specific month in the calendar view. It seems to always be the last photo of the month that is displayed. I've tried to set one as a favorite, but it didn't work. I've waited several days (in case it's a cache issue) and even reloaded my app.

Thanks for your help!

**Comment by graciousgrey** [comment, marked as answer]
Hello :)

Setting cover images using the web ui is planned, see: https://github.com/photoprism/photoprism/issues/383

Until this is implemented you can set a cover image in the database:
- In the albums table set the filehash of the photo you want as cover as "thumb" 
- Set "thumb_src" to "manual" so that your changes do not get overwritten

**Reply by sylvaindubus** [reply]
Hello and thanks for your reply!

This workaround seems a bit cumbersome for multiple albums, I'll patiently wait this implementation instead.

I would be glad to contribute if it can make things going faster somehow. :)
